index.html

```diff
@@ -537,6 +537,67 @@ <h3>Legal Identity</h3>
               area for the duration of the conflict.</dd>
         </dl>
     </section>
+    <section>
+        <h3>Devices</h3>
+        <p>Intelligence devices are created and deployed so that they can interact with other
+        <a>entities</a> (people, organizations, devices). Establishing trust and maintaining
+        secure relationships with these devices is especially critical.</p>
+        <dl class="left dl-horizontal">
+            <dt><udef>I.1 Devices during manufacturing</udef></dt>
+            <dd><p>Bob, the director of production at HVAC Manufacturing, issues a device identifying
+            <a>verifiable credential</a> (e.g. IDevID, IAK) at the factory for an energy-saving
+            fan controller IoT device.</p>
+ 
+            <p>Carol, senior quality engineer at Certifications Testing Lab, issues a certification of specification-compliance <a>verifiable credential</a> to the fan-controller device at
+            the certification lab during the manufacturing process.</p>
+
+            <p>When the fan controller is installed at the customer's office at Modern Office Spaces,
+            the controller's identifying <a>credential</a> can be verified by Sam, IT technician,
+            to establish the identity of the controller as part of the on-boarding of the new
+            controller. The controller's specification-compliance <a>credential</a> is verified
+            to demonstrate the controller's Energy-Star compliance.</p> 
+            </dd>
+            <dt><udef>I.2 Devices during delivery</udef></dt>
+            <dd><p>As the fan controller leaves the factory, additional verifiable <a>credentials</a>
```

Comment:
One of several instances
`verifiable <a>credentials</a>`
-> `<a>verifiable credentials</a>`

Reply:
fixed.